docs/standard/serialization/system-text-json-ignore-properties.md

```diff
@@ -121,6 +121,9 @@ Here's an example object to serialize and JSON output:
 }
 ```
 
+> [!NOTE]
+> The <xref:System.Text.Json.JsonSerializerOptions.IgnoreNullValues> property is deprecated in .NET 5 and later versions, so it isn't shown by IntelliSense. For the current way to ignore null values, see [how to ignore all null-value properties in .NET 5 and later](system-text-json-ignore-properties.md?pivots=dotnet-5-0#ignore-all-null-value-properties).
```

Comment:
```suggestion
> The <xref:System.Text.Json.JsonSerializerOptions.IgnoreNullValues> property is deprecated in .NET 5 and later versions. For the current way to ignore null values, see [how to ignore all null-value properties in .NET 5 and later](system-text-json-ignore-properties.md?pivots=dotnet-5-0#ignore-all-null-value-properties).
```

I assume only obsolete types aren't shown by IntelliSense? And that's cool - good for IntelliSense for not showing those.